about/faq.md

```diff
@@ -23,134 +19,93 @@ platform, as well as different fixes for its supported devices. Quoting the
+
 
 ## What's up with EME? How can I support this feature in my WPE-based product?
 
-There is code in WebKit to support EME but in any case you will need a license
-agreement with DRM CDM providers to access it since this part is not open source. There are three ways you can
-get this working:
+There is code in WebKit to support Encrypted Media Extensions (EME), but in any case, you will need a license agreement with DRM CDM providers to access it, since this part is not open source. There are three ways you can get this working:
+
+- Obtain a license and use the [Thunder OCDM plugin](https://github.com/rdkcentral/Thunder).
+- Write a Thunder-compatible API complement that will work with your DRM system.
+- Write a new CDM backend for WebKit using your DRM system.
 
-- Getting a license and use the [Thunder OCDM plugin](https://github.com/rdkcentral/Thunder).
-- Or write a Thunder compatible API complement that will glue with your DRM system.
-- Or write a new CDM backend for WebKit using your DRM system.
 
-## What is (and isn't) cog?
+## What is (and isn't) Cog?
 
-Quoting [Cog's README](https://github.com/igalia/cog):
+From [Cog's README](https://github.com/igalia/cog):
 
 > Cog is a small single “window” launcher for the WebKit WPE port. It is small,
 > provides no user interface, and is suitable to be used as a Web application
 > container. The “window” may be fullscreen depending on the WPE backend being
 > used.
 
-Cog's usage scenarios span from a MiniBrowser application to a full web-app
-container application meant to run HTML-based User-interfaces on embedded
-platforms and products.
+Cog's usage scenarios span from a MiniBrowser application to a full web-app container application meant to run HTML-based user interfaces on embedded platforms and products.
 
-Although it can run on Linux-based desktop environments, Cog is not a full-blown
-Web Browser to be compared with Google Chrome or Safari. Cog's primary
-environment is on embedded platforms, it can run within a Wayland compositor
-such as Weston. Additionally if the platform supports KMS/DRM, Cog can run as a
-full-screen standalone browser, this use-case is very common on kiosk products
-for instance.
+Although it can run on Linux-based desktop environments, Cog is not a full-blown Web Browser to be compared with Google Chrome or Safari. Cog's primary environment is on embedded platforms, and it can run within a Wayland compositor such as Weston. Additionally, if the platform supports KMS/DRM, Cog can run as a full-screen standalone browser, this use-case is very common on kiosk products for instance.
```

Comment:
Out of curiosity, what does KMS/DRM stand for in this? I don't know what KMS is at all and I would've intuited DRM is digital rights management, but that doesn't make a lot of sense here in this context to me at least.

Reply:
I actually don’t know myself, but I wasn’t sure how much acronym expansion I should be doing in contexts where it was highly likely anyone interested in answer would already know.

Reply:
Turns out in this case:

> KMS stands for Kernel Mode Setting and is a sub-part of the DRM API
> DRM stands for Direct Rendering Manager and was introduced to deal with graphic cards embedding GPUs

(From https://events.static.linuxfound.org/sites/events/files/slides/brezillon-drm-kms.pdf)